src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do List</title>
+  </head>
+  <body>
+    <div class="container">
+      <div id="todo-list" class="todo-list">
```

Comment:
I was thinking that you could change the `div` tags to another syntactic tag like the `article `tag for example